### PR TITLE
[README] Add DebugBundle to the list

### PR DESCRIPTION
**Pull request by edvinasme**
| Q             | A
| ------------- | ---
| Bug fix?      | [no]
| New feature?  | [no]
| BC breaks?    | [no]
| Deprecations? | [no]
| Tests pass?   | [yes]
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

This commits mentions **DebugBundle** in the readme. I believe this should also be ported to 2.7, 2.8 and master branches.

Have a nice weekend!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ It comes pre-configured with the following bundles:
   * [**SensioGeneratorBundle**][13] (in dev/test env) - Adds code generation
     capabilities
 
+  * **DebugBundle** (in dev/test env) - Adds debug and VardDumper component
```
Comment (Pierstoval) [inline README.md]: `VarDumper` instead of `VardDumper` :smile:

Comment (edvinasme) [inline README.md]: Sorry. Fixed in last commit

File: README.md [hunk 2]
```diff
@@ -53,6 +53,9 @@ It comes pre-configured with the following bundles:
   * [**SensioGeneratorBundle**][13] (in dev/test env) - Adds code generation
     capabilities
 
+  * **DebugBundle** (in dev/test env) - Adds debug and VarDumper component
```
Comment (stof) [inline README.md]: the name of the Debug component should have a capital letter too

Comment (edvinasme) [inline README.md]: fixed in the latest commit.

**Comment by edvinasme** [comment]
I have updated it according to comments. If there is anything else - please let me know.

**Comment by fabpot** [comment]
Thank you @edvinasme.

**Pull request closed without merging**